Utility/Utility.Public/Include/Utility/Utility.hpp

```diff
@@ -49,6 +49,16 @@ namespace Utility
         FAILED,
     };
 
+    /**  @brief Enum for tracking channel's subscribe status.
+     *   @details Utility::ChannelSubscribeCodes.
+     */
+    enum class ChannelSubscribingCodes : std::uint8_t
```

Comment:
ChannelSubscribing**Code**

Reply:
I think it should be used in the plural like all the other enum codes in this file